readthedocs/templates/security.txt

```diff
@@ -0,0 +1 @@
+Policy: https://docs.readthedocs.io/en/latest/security.html
```

Comment:
Reading the official page, https://securitytxt.org/ I found that there are more fields we can add here:

```
Contact:
Encryption: 
Acknowledgements: 
Policy: 
Signature: 
Hiring: 
```

Contact, Ecryptation and Signature are good candidates I think.